aws-android-sdk-kinesis-test/src/androidTest/java/com/amazonaws/mobileconnectors/kinesis/kinesisrecorder/KinesisFirehoseRecorderIntegrationTest.java

```diff
@@ -50,15 +50,17 @@ public void setup() throws JSONException {
 
         provider = new CognitoCredentialsProvider(getPackageConfigure().getString("identity_pool_id"),
                 Regions.US_EAST_1);
```

Comment:
This is reasonable to postpone for a future update, but we've been slowly migrating away from hardcoded region values since we want to be able to run integ tests in any region where the service is supported. The `common` stack now includes a `region` key that uses the value of `AWS_DEFAULT_REGION` that was supplied during stack creation.